### DISCUSSION TITLE
Any plans to distribute the app in more formats for Linux users?

### DISCUSSION THREAD

**Discussion by ghost** [strimio/strimio-desktop#12] [Q&A] [answered]
I'm a Linux user but I'm not a Snapcraft user. I was wondering if there's any plans to release the app in more formats in the future (appImages, flathub, etc).

**Comment by spiral2k** [comment, marked as answer]
Hey @fabgrel10, not right now, maybe in the future.

**Comment by Soboku** [comment]
Any chance of an android version?

**Comment by orowith2os** [comment]
If I get time, I can look into repackaging for Flatpak. 

A few things to consider:

- Aarch64/Arm64 binaries
- Build from source, or use binaries from Snapcraft?
- Upstream support for the flatpak binaries
- Will upstream help to maintain the flatpak?
